.rubocop.yml

```diff
@@ -16,5 +16,8 @@ AllCops:
   TargetRubyVersion: 3.2.2
   SuggestExtensions: false
 
+Metrics/MethodLength:
+  Max: 16
+
```

Comment:
A bit of a lazy way to address a RuboCop warning... but also a relatively reasonable allowance for methods length to make.

Reply:
Agreed.